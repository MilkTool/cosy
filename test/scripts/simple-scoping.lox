var x = 3
{
  print x
  var x = 7
  print x
}
