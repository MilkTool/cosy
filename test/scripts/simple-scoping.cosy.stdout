[line 4] Error at "x": Shadows existing variable.
